src/librustc_data_structures/flock.rs

```diff
@@ -32,43 +32,43 @@ mod imp {
         use libc;
 
         #[repr(C)]
-        pub struct flock {
-            pub l_type: libc::c_short,
-            pub l_whence: libc::c_short,
-            pub l_start: libc::off_t,
-            pub l_len: libc::off_t,
-            pub l_pid: libc::pid_t,
+        pub(crate) struct flock {
+            pub(crate) l_type: libc::c_short,
+            pub(crate) l_whence: libc::c_short,
+            pub(crate) l_start: libc::off_t,
+            pub(crate) l_len: libc::off_t,
+            pub(crate) l_pid: libc::pid_t,
 
             // not actually here, but brings in line with freebsd
-            pub l_sysid: libc::c_int,
+            pub(crate) l_sysid: libc::c_int,
         }
 
-        pub const F_RDLCK: libc::c_short = 0;
-        pub const F_WRLCK: libc::c_short = 1;
-        pub const F_UNLCK: libc::c_short = 2;
-        pub const F_SETLK: libc::c_int = 6;
-        pub const F_SETLKW: libc::c_int = 7;
+        pub(crate) const F_RDLCK: libc::c_short = 0;
+        pub(crate) const F_WRLCK: libc::c_short = 1;
+        pub(crate) const F_UNLCK: libc::c_short = 2;
+        pub(crate) const F_SETLK: libc::c_int = 6;
+        pub(crate) const F_SETLKW: libc::c_int = 7;
     }
 
     #[cfg(target_os = "freebsd")]
     mod os {
         use libc;
 
         #[repr(C)]
-        pub struct flock {
-            pub l_start: libc::off_t,
-            pub l_len: libc::off_t,
-            pub l_pid: libc::pid_t,
-            pub l_type: libc::c_short,
-            pub l_whence: libc::c_short,
-            pub l_sysid: libc::c_int,
+        pub(crate) struct flock {
```

Comment:
`pub(super)` would make more sense here and below, since this something in a private module.